packages/rpc-client/test/ws_client_test.ts

```diff
@@ -63,6 +62,7 @@ blockchainTests.resets('WSClient', env => {
             const makerToken = new DummyERC20TokenContract('0x34d402f14d58e001d8efbe6585051bf9706aa064', provider);
             const feeToken = new DummyERC20TokenContract('0xcdb594a32b1cc3479d8746279712c39d18a07fc0', provider);
             const mintAmount = new BigNumber('100e18');
+            // tslint:disable:await-promise
```

Comment:
I'm not sure honestly. At one point TSLint was complaining about awaiting a non-promise type but it might have disappeared since https://github.com/0xProject/0x-mesh/pull/818.